targets/mgprettier/targets.go

```diff
@@ -14,17 +14,22 @@ import (
 
 const packageJSONContent = `{
   "devDependencies": {
-    "prettier": "^2.4.1",
-    "@einride/prettier-config": "^1.2.0"
+    "@einride/prettier-config": "1.2.0",
+    "prettier": "2.5.0"
   }
 }`
 
+const prettierConfigContent = `module.exports = {
+	...require("@einride/prettier-config"),
+}`
+
 var executable string
 
 func FormatMarkdown(ctx context.Context) error {
 	logger := mglog.Logger("prettier")
 	ctx = logr.NewContext(ctx, logger)
-	mg.CtxDeps(ctx, prepare)
+	prettierrc := filepath.Join(mgtool.GetPath(), "prettier", ".prettierrc.js")
+	mg.CtxDeps(ctx, mg.F(prepare, prettierrc))
 	args := []string{
```

Comment:
@filiptammergard I think all that's needed to get it working is to add a flag here that points to the newly written `prettierrc` path

Reply:
That's right! Added it and now it formats markdown correctly 👍

Reply:
Awesome!!